mesonbuild/interpreter.py

```diff
@@ -1207,6 +1208,7 @@ def build_func_dict(self):
                       'add_project_arguments' : self.func_add_project_arguments,
                       'add_global_link_arguments' : self.func_add_global_link_arguments,
                       'add_project_link_arguments' : self.func_add_project_link_arguments,
+                      'add_test_setup' : self.func_add_test_setup,
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent
* [E203] whitespace before ':'

<!-- Comment by [SideCI](https://sideci.com) -->